contents/customers/speakeasy.md

```diff
@@ -0,0 +1,65 @@
+---
+title: How Speakeasy manages features and developer relations with PostHog
```

Comment:
Should be 'using PostHog' rather than 'with PostHog'?

Reply:
```suggestion
title: How Speakeasy manages features and developer relations using PostHog
```